packages/visual-editor/scripts/generateTranslations.js

```diff
@@ -0,0 +1,154 @@
+import fs from "fs/promises";
+import fetch from "node-fetch";
+import pLimit from "p-limit";
+
+const defaultLng = "en";
+
+// languages codes from Yext account settings
+const targetLngs = [
+  "zh_CN",
+  "zh_TW",
+  "hr_HR",
+  "cs_CZ",
+  "da_DK",
+  "nl",
+  "en_GB",
+  "en_US",
+  "et_EE",
+  "fi_FI",
+  "fr_FR",
+  "de_DE",
+  "hu_HU",
+  "it_IT",
+  "ja_JP",
+  "lv_LV",
+  "lt_LT",
+  "nb_NO",
+  "pl_PL",
+  "pt_PT",
+  "ro_RO",
+  "sk_SK",
+  "es_ES",
+  "sv_SE",
+  "tr_TR",
+];
+
+// translates Yext locales to Google translate compatible locales
+const localeToGoogleTranslateMap = {
+  zh_CN: "zh-CN",
+  zh_TW: "zh-TW",
+  hr_HR: "hr",
+  cs_CZ: "cs",
+  da_DK: "da",
+  nl: "nl",
+  en_GB: "en",
```

Comment:
This api doesn't have dialects? Not amazing but ok. Like for British English some words are spelled differently: `favor` vs `favour` as an example.